app/workers/search/remove_from_elasticsearch_index_worker.rb

```diff
@@ -6,6 +6,10 @@ class RemoveFromElasticsearchIndexWorker
 
     def perform(search_class, id)
       search_class.safe_constantize.delete_document(id)
+    rescue Search::Errors::Transport::NotFound => e
```

Comment:
Can we add a comment explaining why we have to do this? Maybe something like:
```ruby
# Reactions are often destroyed before the indexing job can execute so we ignore this error
```
It might help our future selves remember why we did this. Or if we end up having to rescue for other document types, we'll know why for each one.

Reply:
added to both workers actually.